### PR TITLE
Can no longer summon dead cultists

### PR DESCRIPTION
**Pull request by Robustin**
Considering they can be revived with little effort, this has to be stopped to prevent them from becoming unkillable.

### CODE REVIEW

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -780,6 +780,11 @@ var/list/teleport_runes = list()
 		fail_invoke()
 		log_game("Summon Cultist rune failed - target in away mission")
 		return
+	if(!isliving(cultist_to_summon))
```
Comment (KorPhaeron) [inline code/game/gamemodes/cult/runes.dm]: isliving checks to see if they're a /mob/living, not if they're alive

**Comment by Eaglendia** [comment]
dude just decapitate them lmao

File: code/game/gamemodes/cult/runes.dm (hunk 1 above)
Comment (Robustin) [inline code/game/gamemodes/cult/runes.dm]: god I knew it wouldn't be that simple, well at least I opened it before the feature freeze

Comment (KorPhaeron) [inline code/game/gamemodes/cult/runes.dm]: Joan just did the same thing in another PR, saves you the trouble entirely

**Pull request closed without merging**